quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -410,6 +392,67 @@ impl IndexingService {
         Ok(indexes_metadata)
     }
 
+    /// Fetches the immature splits candidates for merge for all the indexing pipelines for which a
+    /// merge pipeline is not running.
+    async fn fetch_immature_splits(
```

Comment:
can we change the name to express the fact this is target all new pipelines?

(it has the same name as the individual method in the merge pipeline here)